### PR TITLE
Couple more packages

### PR DESCRIPTION
**Pull request by z3ntu**
indicator-messages & ubuntu-push

No idea if the ubuntu-push client actually works on Arch Linux, it definitely crashes without accountsservice-ubuntu running - but at least everything compiles :) Patches are also submitted upstream (except `0004-cual-link-against-ual-3.patch`)

### CODE REVIEW

File: indicator-messages-git/PKGBUILD [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Maintainer: Ivan Semkin (ivan at semkin dot ru)
```
Comment (vanyasem) [inline indicator-messages-git/PKGBUILD]: this change made my day

File: ubuntu-download-manager-git/PKGBUILD [hunk 2]
```diff
@@ -12,7 +12,7 @@ conflicts=(ubuntu-download-manager)
 provides=(ubuntu-download-manager)
 depends=(boost dbus-test-runner gmock qt5-base qt5-declarative libnih google-glog)
 makedepends=(git doxygen cmake cmake-extras-git gcovr lcov)
-source=('git+https://github.com/ubports/ubuntu-download-manager.git'
+source=('git+https://github.com/ubports/ubuntu-download-manager.git#branch=xenial'
```
Comment (vanyasem) [inline ubuntu-download-manager-git/PKGBUILD]: I totally forgot that you can do that, thanks

**Comment by vanyasem** [comment]
Thank you! Looks good

File: indicator-messages-git/PKGBUILD (hunk 1 above)
Comment (z3ntu) [reply indicator-messages-git/PKGBUILD]: oops forgot to squash that :D

**Pull request merged**